### PR TITLE
[Improvement][install] The new shell script is used to view the server processes of nodes #3347

### PR DESCRIPTION
**Pull request by zhuangchong**
## What is the purpose of the pull request

add status-all.sh,used to view the server processes of nodes, similar to start- all.sh

## Brief change log

*(for example:)*
status-all.sh
install.sh
dolphinscheduler-dist/pom.xml

dolphinscheduler-dist/src\main\assembly\dolphinscheduler-binary.xml
dolphinscheduler-dist/src\main\assembly\dolphinscheduler-nginx.xml

## Verify this pull request

*(Please pick either of the following options)*

This change added tests and can be verified as follows:

*(example:)*
  - *Manually verified the change by testing locally.*
add status-all.sh,used to view the server processes of nodes, similar to start- all.sh

### CODE REVIEW

**Comment by yangyichao-mango** [comment]
Hi,
This is a new feature, and please create an issue for this new feature.

**Comment by zhuangchong** [comment]
> create an issue for this new feature.

A new one has been created Improvement #3347，Whether it can be or not?

File: script/status-all.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+workDir=`cd ${workDir};pwd`
+source $workDir/../conf/config/install_config.conf
+
+echo -e '\n'
+echo "====================== dolphinscheduler install config============================="
+echo -e "1.dolphinscheduler server node install hosts:[ \033[1;32m ${ips} \033[0m ]"
+echo -e "2.master server node install hosts:[ \033[1;32m ${masters} \033[0m ]"
+echo -e "3.worker server node install hosts:[ \033[1;32m ${workers} \033[0m ]"
+echo -e "4.alert server node install hosts:[ \033[1;32m ${alertServer} \033[0m ]"
+echo -e "5.api server node install hosts:[ \033[1;32m ${apiServers} \033[0m ]"
```
Comment (dailidong) [inline script/status-all.sh:29]: what these "[ \033[1;32m" means ?

Comment (zhuangchong) [reply script/status-all.sh:29]: \033[1;32m xxx \033[0m  It indicates that the color of the middle text is currently green. Do you want to remove it?

---
\033[1;32m xxx \033[0m 表示打印中间文本颜色的，目前是绿色，需要去掉么？

Comment (dailidong) [reply script/status-all.sh:29]: I take your point, no need to remove

**Pull request merged**